lib/gtfs_realtime_viz.ex

```diff
@@ -36,16 +36,32 @@ defmodule GTFSRealtimeViz do
   Renders the received protobuf files and comments into an HTML fragment that can either
   be opened directly in a browser or embedded within the HTML layout of another app.
   """
-  @spec visualize(term) :: String.t
-  def visualize(group) do
-    [vehicle_archive: vehicles_by_stop_id(group), routes: routes()]
+  @spec visualize(term, %{String.t => [[String.t]]}) :: String.t
```

Comment:
Can you update the doc comment for this function to explain what the `opts` needs to look like? That will make the generated documentation more useful.